### DISCUSSION TITLE
Chinese Translation

### DISCUSSION THREAD

**Discussion by dannytszho** [TandoorRecipes/recipes#791] [Ideas]
Hi Vabene,

I really like the idea of your recipes App. I am new to programming and I really want to help with your project. My programming skills are pretty limited, I might only be able to help with Front-end stuff. I am fluent in Mandarin and Cantonese, I think I can mostly focus on the translation of the App. I saw you only have a Simplified Chinese version on Weblate. I am wondering if you want to do both traditional Chinese and Simplified Chinese? I am looking forward to hearing from you.

Best,
Danny

**Comment by vabene1111** [comment]
I honestly don't know the differences but more languages are always welcome. As far as I know you can simply add whatever language you desire, if not feel free to send me the language codes so that I don't mix them up and I will add them.

Also happy that you like the app and want to contribute, thank you very much

**Reply by dannytszho** [reply]
No problem man. I also have a question. I pulled the latest version of images using Docker yesterday. And when I ran the App on my local server, it still runs with the old search on the search page, when I looked at the code on models.py. The code is different from the one on Github. I am wondering if you have committed the latest version in Docker? OR just me pulled the old one?

**Reply by vabene1111** [reply]
there are different tags on docker, by default you pull `latest` which is synced with the master branch. You can use `develop` but that is broken from time to time so use at your own risk. 

The new search is however already released on master, its just not the default layout. You can switch to it using the settings "search style". From the next release on which will be released soon the new search will be the default and the old one will eventually be deprecated

**Reply by dannytszho** [reply]
Got it! Thank you for clarifying.
